packages/vm/tests/api/EIPs/eip-2537-BLS.spec.ts

```diff
@@ -75,4 +76,31 @@ tape('EIP-2537 BLS tests', (t) => {
+        'hex'
+      ),
+      gasLimit: new BN(5000000),
+      _common: common,
+      _VM: vm,
+    })
+
+    const testVectorResult =
+      '0x00000000000000000000000000000000083ad744b34f6393bc983222b004657494232c5d9fbc978d76e2377a28a34c4528da5d91cbc0977dc953397a6d21eca20000000000000000000000000000000015aec6526e151cf5b8403353517dfb9a162087a698b71f32b266d3c5c936a83975d5567c25b3a5994042ec1379c8e526000000000000000000000000000000000e3647185d1a20efad19f975729908840dc33909a583600f7915025f906aef9c022fd34e618170b11178aaa824ae36b300000000000000000000000000000000159576d1d53f6cd12c39d651697e11798321f17cd287118d7ebeabf68281bc03109ee103ee8ef2ef93c71dd1dcbaf1e0'
```

Comment:
nice, could you add a little explanation of what this is actually testing and what the expected result represents?

Reply:
Will do my best. I'd be lying if I said I really understood the details of the math but I can try. At this point, I basically just understand that this is one of the operations used in verifying BLS signatures and the input and output strings are just test data verified to work by matter labs.

Reply:
haha ok makes sense, yeah even just some info like that and a link to the file in OP would be good, I'm just thinking if someone wants to debug an issue there in the future it'd be helpful to at least have some context of what it is.